package.json

```diff
@@ -1,19 +1,20 @@
 {
   "name": "@graphprotocol/contracts",
-  "version": "5.3.0",
+  "version": "5.3.2",
```

Comment:
nit: why not 5.3.1?

Reply:
I think 5.3.1 was claimed and published from another branch.

Reply:
Oh, looks like main is at 5.3.3 already, so I suppose this'll be 5.3.4 after rebasing